products.js

```diff
@@ -29,15 +29,15 @@ const VIEWPORT_HEIGHT = 960;
  * Remove dollar sign, strip whitespace, strip words (anything not numeric or a price symbol), and remove trailing zeros
  */
 function formatPrice(priceString){
-  priceString = priceString.replace('$', '').replace(/([\s]|[^0-9$.-])/g, '').replace(/(\.[0-9]*?)0+$/, '');
```

Comment:
Is this change on purpose? (It wasn't mentioned in the commit message, and I can't tell offhand what it does, so I'm not sure.)